{
  "$schema": "http://cyclonedx.org/schema/bom-1.6.schema.json",
  "bomFormat": "CycloneDX",
  "specVersion": "1.6",
  "serialNumber": "urn:uuid:ea696f99-3aff-4365-8abe-6e7485ccf432",
  "version": 1,
  "components": [
    {
      "bom-ref": "pkg:generic/haxx/curl@7.82.0",
      "cpe": "cpe:2.3:a:haxx:curl:7.82.0:*:*:*:*:*:*:*",
      "group": "haxx",
      "name": "curl",
      "purl": "pkg:generic/haxx/curl@7.82.0",
      "type": "library",
      "version": "7.82.0"
    }
  ],
  "vulnerabilities": [
    {
      "advisories": [
        { "url": "https://curl.se/docs/CVE-2024-2398.html" }
      ],
      "affects": [
        { "ref": "pkg:generic/haxx/curl@7.82.0" }
      ],
      "analysis": {
        "detail": "Our team is doing the review",
        "state": "in_triage"
      },
      "bom-ref": "CVE-2024-2398",
      "description": "When an application tells libcurl it wants to allow HTTP/2 server push, and the amount of received headers for the push surpasses the maximum allowed limit (1000), libcurl aborts the server push. When aborting, libcurl inadvertently does not free all the previously allocated headers and instead leaks the memory.  Further, this error condition fails silently and is therefore not easily detected by an application.",
      "id": "CVE-2024-2398",
      "ratings": [
        {
          "method": "CVSSv31",
          "score": "8.6",
          "severity": "high",
          "source": {
            "name": "134c704f-9b21-4f2e-91b3-4a467353bcc0"
          },
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:L/A:L"
        }
      ],
      "source": {
        "name": "NVD",
        "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-2398"
      }
    }
  ]
}
